src/TestingUtils/Microsoft.AspNetCore.Testing/test/MaximumOSVersionTest.cs

```diff
@@ -42,7 +42,7 @@ public void RunTest_Win10_RS4()
             Assert.NotNull(versionKey);
             var currentVersion = (string)versionKey.GetValue("CurrentBuildNumber");
             Assert.NotNull(currentVersion);
-            Assert.True(17134 >= int.Parse(currentVersion));
+            Assert.True(17134 >= int.Parse(currentVersion), $"Unexpected build number {currentVersion} on {Environment.OSVersion}.");
```

Comment:
```suggestion
            Assert.True(17134 >= int.Parse(currentVersion), $"Unexpected build number {currentVersion} on {Environment.OSVersion.Version}.");
```
https://github.com/dotnet/extensions/blob/4ac4b19372773dc7a226309497495dcc26d9e26b/src/TestingUtils/Microsoft.AspNetCore.Testing/src/xunit/MaximumOSVersionAttribute.cs#L74